x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/source_logic.ts

```diff
@@ -209,7 +236,9 @@ export const SourceLogic = kea<MakeLogicType<SourceValues, SourceActions>>({
         const response = await HttpLogic.values.http.patch(route, {
           body: JSON.stringify({ content_source: source }),
         });
-        actions.onUpdateSourceName(response.name);
+        if (source.name) {
```

Comment:
Just for my own understanding, when is there _not_ a source name?

Reply:
The `updateContentSource` method accepts partial source patch payloads, so this logic is just checking to see if the `name` was sent to this method call.